base/src/commonTest/kotlin/state/StateRepoTest.kt

```diff
@@ -117,43 +120,49 @@ class StateRepoTest: FlowableTest<TrafficLightState>() {
     }
 
     @Test
-    fun changeState() = runBlockingWithFlow {
-        lateinit var greenState: TrafficLightState.GreenLight
-        test {
+    fun testChangeState() = testWithFlow {flowTest ->
+        val greenStateDeferred = CompletableDeferred<TrafficLightState.GreenLight>()
+        flowTest.test {
```

Comment:
We can expose the methods of `flowTest` by making it the implicit this. So we can keep writing `test{` instead of `flowTest.test` but have it act the same otherwise.

https://kotlinlang.org/docs/reference/lambdas.html#function-literals-with-receiver